applications/repertoire/templates/_helpers.tpl

```diff
@@ -11,6 +11,9 @@ Common labels
 {{- define "repertoire.labels" -}}
 helm.sh/chart: {{ include "repertoire.chart" . }}
 {{ include "repertoire.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
```

Comment:
I've been removing this setting everywhere since it creates a lot of changes each time the app version changes that clutter up the diff but don't really change anything.